extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -312,16 +295,13 @@ const OHIFCornerstoneViewport = React.memo(
         return;
       }
 
-      const unsubscribeFromJumpToMeasurementEvents = _subscribeToJumpToMeasurementEvents(
-        elementRef,
-        viewportId,
-        servicesManager
+      const { unsubscribe } = measurementService.subscribe(
```

Comment:
Can we call it jumpToViewReference?  It just feels like there should be uses of it for things other than measurements - I'm thinking segmentation changes or remembering previously displayed layouts on layout change?